tests/src/Interop/Desktop/PInvokeTest/Attributes/LCID/LCIDNative.cpp

```diff
@@ -0,0 +1,146 @@
+﻿#include <stdio.h>
+#include <windows.h>
+#include <tchar.h>
+
+char* strManaged = "Managed\0String\0";
+int   lenstrManaged = 7; // the length of strManaged
+
+char* strReturn = "a\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0";
+char* strFalseReturn = "\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0";
+
+char* strNative = " Native\0String\0";
+int lenstrNative = 7; //the len of strNative
+
+//Test Method1
+
+extern "C" LPSTR ReturnString()
+{
+    size_t lenstrReturn = strlen(strReturn);
+    LPSTR ret = (LPSTR)(CoTaskMemAlloc(sizeof(char)*(lenstrReturn+1)));
+    memset(ret,'\0',lenstrReturn+1);
+    strncpy_s(ret,lenstrReturn + 1,strReturn,1);
+    return ret;
+}
+
+extern "C" LPSTR ReturnFalseString()
+{
+    size_t lenstrFalseReturn = strlen(strFalseReturn);
+    LPSTR ret = (LPSTR)(CoTaskMemAlloc(sizeof(char)*(lenstrFalseReturn+1)));
+    memset(ret,'\0',lenstrFalseReturn+1);
+    strncpy_s(ret,lenstrFalseReturn + 1,strFalseReturn,1);
+    return ret;
+}
+
+//Test Method2
+extern "C" __declspec(dllexport) LPSTR Marshal_InOut1(int lcid, LPSTR s)
```

Comment:
Look for `DLLEXPORT` and you should include `<xplatform.h>` - see existing tests.

Reply:
Thanks, should be fixed